packages/deployment/app/config.yml

```diff
@@ -149,12 +149,17 @@ default:
       state: ENABLED
       targets:
         - lambda: jobs
-    cronJobs:
+    dailyEmsIngestReport:
       # run at 5:00am GMT daily
       schedule: cron(0 5 * * ? *)
       state: ENABLED
       targets:
         - lambda: EmsReport
+    dailyEmsDistributionReport:
+      schedule: cron(0 5 * * ? *)
+      state: ENABLED
+      targets:
+        - lambda: EmsDistributionReport
```

Comment:
Could this have just been added as an extra line to the job above under lambdas?

Reply:
+1 I think adding extra line to existing cronJob should work.

Reply:
I tried that first but discovered that having multiple targets causes the deployment to fail.  I talked to @scisco about it and decided that the easiest thing would be to just create a new scheduled task.  Doing it as a separate scheduled task also allows this report to be generated at a different time than the ingest report.

Reply:
👍